packages/devprod-status-bot/src/index.ts

```diff
@@ -639,7 +858,16 @@ export default {
 			});
 		}
 
-		return new Response("");
+		return new Response(
+			JSON.stringify({
+				success: true,
+				message: "Request processed",
+			}),
+			{
+				status: 200,
+				headers: { "Content-Type": "application/json" },
+			}
+		);
```

Comment:
I don't think this is needed. Github webhooks just care about the status code being 2xx